psydac/fem/basic.py

```diff
@@ -114,6 +119,42 @@ def eval_field_gradient( self, field, *eta , weights=None):
 
         """
 
+    # ...
+    def __mul__(self, a):
+
+        if self.is_product:
+            spaces = list(self.spaces)
+        else:
+            spaces = [self]
+
+        if a.is_product:
+            spaces += a.spaces
+        else:
+            spaces += [a]
+
+        space = ProductFemSpace(*spaces)
+        if a.symbolic_space and self.symbolic_space:
+            space._symbolic_space =  self.symbolic_space*a.symbolic_space
+        return space
+
+    # ...
+    def __rmul__(self, a):
+        if a.is_product:
+            spaces = list(a.spaces)
+        else:
+            spaces = [a]
+
+        if self.is_product:
+            spaces += list(self.spaces)
+        else:
+            spaces += [self]
+
+        space = ProductFemSpace(*spaces)
+
+        if a.symbolic_space and self.symbolic_space:
+            space._symbolic_space =  a.symbolic_space * self.symbolic_space
+        return space
```

Comment:
Please note that methods `__mul__` and `__rmul__` are not `abstractmethod`s, and therefore they do not belong to the "abstract interface" section. So they should be moved to another section, which I would call "Concrete methods"

Reply:
done